configs/pare/hrnet_w32_conv_pare_coco_cache.py

```diff
@@ -1,15 +1,16 @@
+_base_ = ['../_base_/default_runtime.py']
 use_adversarial_train = True
 
 # evaluate
 evaluation = dict(interval=10, metric=['pa-mpjpe', 'mpjpe'])
-# optimizer
 
+img_res = 224
+
+# optimizer
 optimizer = dict(
-    backbone=dict(type='Adam', lr=2.0e-4),
-    head=dict(type='Adam', lr=2.0e-4),
-)
+    backbone=dict(type='Adam', lr=2.0e-4), head=dict(type='Adam', lr=2.0e-4), )
```

Comment:
This line is too long and might cause the lint error, the same with other configs.